project/AppDependencies.scala

```diff
@@ -1,29 +1,32 @@
 import sbt._
 
 object AppDependencies {
-  import play.core.PlayVersion
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo"   %% "play2-reactivemongo"            % "0.20.13-play28",
-    "uk.gov.hmrc"         %% "play-frontend-hmrc"             % "3.6.0-play-28",
+    "org.reactivemongo"   %% "play2-reactivemongo"            % "1.1.0-play28-RC4",
+    "uk.gov.hmrc"         %% "play-frontend-hmrc"             % "3.22.0-play-28",
     "uk.gov.hmrc"         %% "play-conditional-form-mapping"  % "1.11.0-play-28",
-    "uk.gov.hmrc"         %% "domain"                         % "8.0.0-play-28",
-    "uk.gov.hmrc"         %% "emailaddress"                   % "3.5.0",
-    "uk.gov.hmrc"         %% "bootstrap-frontend-play-28"     % "5.21.0",
-    "com.typesafe.play"   %% "play-json-joda"                 % "2.7.4"
+    "uk.gov.hmrc"         %% "domain"                         % "8.1.0-play-28",
+    "uk.gov.hmrc"         %% "emailaddress"                   % "3.6.0",
+    "uk.gov.hmrc"         %% "bootstrap-frontend-play-28"     % "6.3.0",
+    "com.typesafe.play"   %% "play-json-joda"                 % "2.9.2"
  )
 
   val test: Seq[ModuleID] = Seq(
-    "org.scalatest"            %% "scalatest"             % "3.0.9",
-    "org.scalatestplus.play"   %% "scalatestplus-play"    % "5.0.0",
+    "org.scalatest"            %% "scalatest"             % "3.2.12",
+    "org.scalatestplus.play"   %% "scalatestplus-play"    % "5.1.0",
+    "org.scalatestplus"        %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
     "org.pegdown"              %  "pegdown"               % "1.6.0",
-    "org.jsoup"                %  "jsoup"                 % "1.10.3",
-    "com.typesafe.play"        %% "play-test"             % PlayVersion.current,
```

Comment:
This should be ok to leave as "PlayVersion.current" for the version. It gets pulled from the version of the "sbt-plugin" in plugins.sbt (which can be updated to 2.8.16)

Reply:
Updated this now